uint16 gain
---
bool success
